en/docs/try-out/account-information-service-flow.md

```diff
@@ -0,0 +1,896 @@
+
+10. Click **Save**.
+
+11. Select **Runtime** from the left menu pane.
+
+12. Toggle the **Schema Validation** button to enable Schema Validation. ![schema-validation](../assets/img/get-started/quick-start-guide/schema-validation.png)
+
+13. Click the **Edit** button under **Request > Message Mediation**. ![edit_message_mediation](../assets/img/get-started/quick-start-guide/edit-message-mediation.png)
+
+14. Select the **Custom Policy** option.
+
+15. Upload the `<APIM_HOME>/<OB_APIM_TOOLKIT_HOME>/repository/resources/apis/berlin.group.org/PSD2API_1.3.6/dynamic-endpoint-insequence-1.3.6.xml` insequence file. ![uplaoad_insequence](../assets/img/try-out/account-flow/upload-insequence.png)
+ 
+16. Click **Select**. 
+
+17. Scroll down and click **SAVE**.
+
+18. Use the left menu panel and go to **API Configurations > Endpoints**. 
+
+    ![select_endpoints](../assets/img/get-started/quick-start-guide/select-endpoints.png)
+
+19. Add a **Dynamic Endpoint**. ![add_dynamic_endpoint](../assets/img/get-started/quick-start-guide/add_dynamic_endpoint.png)
+
+20. Go to **Deployments** using the left menu pane. 
+
+    ![select_deployments](../assets/img/get-started/quick-start-guide/select-deployments.png)
+    
+21. Select the API Gateway type, in this scenario, it is **Default**. ![api_gateway](../assets/img/get-started/quick-start-guide/dcr-api-gateway.png)
+
+22. Click **Deploy**.
+
+23. Go to **Overview** using the left menu pane. 
+
+    ![select_overview](../assets/img/get-started/quick-start-guide/select-overview.png)
+
+24. Click **Publish**. ![publish_api](../assets/img/get-started/quick-start-guide/publish-api.png)
+
+## Subscribing to PSD2 API
+
+1. The deployed API is now available in the Developer Portal at `https://<APIM_HOST>:9443/devportal`.
+
+2. Select the **NextGenPsd2XS2AFramework v1** API.
+ 
+3. Locate **Subscriptions** from the left menu pane. 
+
+    ![select_subscriptions](../assets/img/get-started/quick-start-guide/select-subscriptions.png)
+    
+4. From the **Application** dropdown, select the application that you want to be subscribed to the NextGenPsd2XS2AFramework v1 API. ![subscribe_to_api](../assets/img/try-out/account-flow/subscribe-to-api.png)
+
+5. Click **Subscribe**.
+
+## Invoking PSD2 API
+
+### Generating application access token
+
+Once you register the application, generate an application access token.
+
+1. Generate the client assertion by signing the following JSON payload using supported algorithms. 
+
+    !!! note
+        If you have configured the [OB certificates](https://openbanking.atlassian.net/wiki/spaces/DZ/pages/252018873/OB+Root+and+Issuing+Certificates+for+Sandbox), 
+        download the certificate and keys attached [here](../../assets/attachments/Certificates.zip), and use them for signing and transports layer security testing purposes.
```

Comment:
Im assuming these are the certs used for UK. But these needs to be changed. Since we have only self-signed certs for Berlin we need to see which certs can be added here.

Reply:
@Akila94

Reply:
Fixed in 492e0e859d9d73f0ff9ad20698d7e1ca2e26bf9a & 69359a31b168e4c48b49c82cf618431b74398042